CHARTER.md

```diff
@@ -10,33 +10,37 @@ The Arcalot Round Table *(ART)* consists of:
 * 4 chairs with equal rights who
   - Enforce the [Code of Conduct](https://github.com/arcalot/.github/blob/main/CODE_OF_CONDUCT.md)
   - Have administrative rights over the [GitHub org](https://github.com/arcalot), communication channels, and other community-owned resources
-  - Publicly record decisions made by the ART in the [art-decisions](art-decisions) folder
+  - Publicly record decisions made by the ART in the [`art-decisions`](art-decisions) folder
 * Individuals who want to join the ART and pass a review of their PR sent to [ART_MEMBERS.md](ART_MEMBERS.md), with approval depending on any one or more of the below requirements:
   - They contribute with code, documentation, artwork, articles, videos, talks, or similar additions to the Arcalot ecosystem
   - They are an active user of Arcalot projects (end user, product builder, researcher, etc. - please describe your use case in the PR, you do not have to mention any company names or details)
   - They are actively helping community members on any of the official Arcalot channels
 
-### Termination of ART membership
-
-An Arcalot Round Table member *(including chairs)* that does not participate in round table matters (voting) *without reason and notice* for 3 times in a row leaves the round table automatically as we can assume they are not interested in being a member of the ART anymore. This ensures that we do not have a corpus of inactive members with voting rights.
+### ART Participation
 
 Meetings do not count as necessary round table matters as we want to include people in all stages of life and across all timezones. We therefore refrain from any synchronous or undocumented decision making.
 
 Additionally, as stated in the [Arcalot Code of Conduct](https://github.com/arcalot/.github/blob/main/CODE_OF_CONDUCT.md), chairs can remove an ART member if their behavior is clearly disruptive to the community.
 
+
+### Chair Participation
+
+Chairs are expected to remain actively involved in the community. An Arcalot Round Table chair that does not participate in round table matters (voting) *without reason and notice* for 3 times in a row leaves the chairs automatically as we can assume they are not interested in the responsibilities of the chair position anymore.
```

Comment:
I think this could benefit from splitting the sentence and/or removing the assumption. They simply did not fulfill their duties, so they are demoted to member.

Reply:
"round table matters (voting)" sounds designed to be obscure. If "matters" means "voting", then can we just say "round table voting" (or to be more specific in the context of our "ART", maybe just "ART voting"?

Reply:
Spelling of numbers is always a strange English convention to figure out. A lone small number being spelled out I agree makes sense, but if there is a larger number in proximity to that number, they should be consistent, likely meaning they should both be Arabic numerals. And at what point is it appropriate to adopt a consistent style among a portion of a document or an entire document? I'm sure this is covered in multiple conflicting style guides.

Reply:
Yeah, the "spelling of numbers" rule is fine for casual reading; I'm less inclined to consider that for technical documents, and the crossover point can be squishy. In *this* context, we don't use "a lot" of numbers, but there are more than a few and I agree that sometimes consistency in form is more important.

Reply:
> I'm sure this is covered in multiple conflicting style guides.

I commend to you _Strunk and White_...any style guides which conflict with that are wrong.  🙃

Reply:
Forget it. I'm re-writing this whole thing in German. :stuck_out_tongue_closed_eyes: